src/app/index.ts

```diff
@@ -266,10 +232,9 @@ module.exports = yo.extend({
     return new Promise((resolve, reject) => {
       try {
         let language = this.project.scriptType === typescript ? 'ts' : 'js';
-        const starterCode = generateStarterCode(this.project.host);
-        const templateFills = Object.assign({}, this.project, starterCode);
         let jsonData = new projectsJsonData(this.templatePath());
         let projectRepoBranchInfo = jsonData.getProjectRepoAndBranch(this.project.projectType, language);
+        const templateFills = Object.assign({}, this.project);
```

Comment:
Is this still needed (for manifest only)?

Reply:
Yes, we are using this to set the manifest guid. Do you think we shoud just use modifyManifestFile for this? The reason I didn't use modifyManifestFile is that this restores the manifest to an imperfect state with comments removed

Reply:
It's fine. Ideally we would fix modifyManifestFile to preserve comments, but this is fine for now.